notebooks/03-gravity/fwd_gravity_anomaly_3d.ipynb

```diff
@@ -609,6 +611,7 @@
     "    rhoMap=model_map,\n",
     "    ind_active=active_cells,\n",
     "    store_sensitivities=\"forward_only\",\n",
+    "    engine=\"choclo\",\n",
```

Comment:
For introductory tutorials like this, I do not want to use functionality that comes from soft dependencies. This notebook should run with basic simpeg. I think we should:
* expose the `engine` argument but set it to the default value
* reword the markdown talking about the choclo engine accordingly.

Reply:
Makes sense! I'll apply these changes then.

Reply:
Looks great!